Added Cloud.yaml file that will create and push the docker image for the repo to quay.io org

ayesha-kr opened the pull request:
Associated issue: ortelius/ortelius#114
 Some other minor fixes.
  -  Updated the image path in the helm chart by pointing it to Quay hosted image. 
  -  Updated the Readiness and liveness probe.

Commit:
Added cloud.yaml file to push the docker image to quay.io organization.

Commit:
updated file path.

markjacksonfishing reviewed (changes requested):
This code looks fine but what is missing are:
* A corresponding issue describing what the problem that needs to be fixed is
* A better description in the PR that describes:
   - A title
   - A detailed description of what the PR is for, does, and issue it solved

Commit:
Added Cloud.yaml file that will create and push the docker image for this repo to quay.io org.

Associated issue: https://github.com/ortelius/ortelius/issues/114

Some other minor fixes.
Updated the image path in the helm chart by pointing it to Quay hosted image.
Updated the Readiness and liveness probe.

Commit:
Waiteress-server has some CVEs that we can't fix.
Hence removed the waitress code serving part from the code
Added a new server https://github.com/tiangolo/meinheld-gunicorn-flask-docker

Now the app while running from docker will be serverd by gunicorn

Commit:
Updated read me

ayesha-kr committed:
Update README.md
